src/styles/components/forms.scss

```diff
@@ -1,50 +0,0 @@
-@import 'variables';
-
```

Comment:
Why is this file being deleted?  Is form styling no longer necessary or being done somewhere else?